.eslintrc

```diff
@@ -1,6 +1,9 @@
 {
   "plugins": ["patternfly-react"],
-  "extends": ["plugin:patternfly-react/recommended"],
+  "extends": [
+    "plugin:patternfly-react/recommended",
+    "./node_modules/@theforeman/vendor-dev/eslint.extends.js"
```

Comment:
just wondering, shouldn't this end up being a plugin just like PF?

Reply:
Yes :)
I think it should be delivered with `@theforeman/env`